app/bundles/LeadBundle/Helper/EventHelper.php

```diff
@@ -41,6 +41,23 @@ public static function updateTags(Lead $lead, $config, MauticFactory $factory)
      * @param $action
      * @param $factory
      */
+    public static function RemoveDoNotContact($action, $factory, $config)
+    {
+        /** @var \Mautic\LeadBundle\Model\LeadModel $leadModel */
+        $leadModel = $factory->getModel('lead');
+        $emailRepo = $factory->getModel('email');
+        $lead      = $leadModel->getCurrentLead();
+        $email_address = $lead->getEmail();
```

Comment:
This is really minor, but all variables in Mautic are camelCased. Could you rename it to $emailAddress?